ccxt_backtesting_exchange/backtester.py

```diff
@@ -223,12 +231,60 @@ def _update_asset_balance(self, asset: str, column: str, amount: float) -> None:
         # Update the balance by adding/subtracting the amount
         self.__update_df_value_by_column(self._balances, "asset", asset, column, amount)
 
+    def fill_orders(self):
+        """
+        Fill orders that are fillable on the current timestamp.
+        """
+        open_orders = self._orders[self._orders["status"] == OrderStatus.OPEN.value]
+        symbols = open_orders["symbol"].unique()
+
+        for symbol in symbols:
+            [timestamp, open, high, low, close, volume] = self._data_feeds[
+                symbol
+            ].get_data_at_timestamp(self.milliseconds())
+            base_asset, quote_asset = symbol.split("/")
+            for index, order in open_orders[open_orders["symbol"] == symbol].iterrows():
+                trade_value = order["amount"] * order["price"]
+                if order["price"] >= low and order["price"] <= high:
```

Comment:
@amustapha Should we not be using the open/close price instead of checking if the order was within the range? On a second though using the range seems a bit more valid because that shows the order must have executed within that timeframe, though we can't tell when exactly. I guess this should be fine.

Reply:
exactly. the only con is, if price is exactly low, we might have a patial fill and that's hard to predetermine. using > or < explicitly would also be inaccurate.